index.html

```diff
@@ -5,8 +5,92 @@
     <title>HTML Academy: Техномарт</title>
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS, уровень 1</a>».</p>
-
+<header class="header-main">
+<p class="header-logo"><img src="https://via.placeholder.com/111x18" alt="Логотип «Техномарта»" width="111" height="18"></p>
+<!-- Здесь будет форма поиска -->
+<p class="header-bookmarks"><a href="#">Закладки: 0</a></p>
+<p class="header-cart"><a href="#">Корзина: 0</a></p>
+<p class="header-order"><a href="#">Оформить заказ</a></p>
+<p class="header-description">Интернет-магазин<br>
+строительных материалов<br>
+и инструментов для ремонта</p>
+<div class="header-contacts">
+<span class="header-phone">+7 (812) 555-05-55</span>
+<p class="header-address">г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</p></div>
+<div class="user-navigation">
+<ul class="user-navigation-unauthorized"
```

Comment:
Пропущена закрывающая скобка тега, отсюда ошибка в валидаторе и некорректное отображение пункта меню
```suggestion
<ul class="user-navigation-unauthorized">
```